mod/admin.php

```diff
@@ -720,7 +709,7 @@ function admin_page_summary(App $a)
 	if (Config::get('system', 'check_new_version_url', 'none') != 'none') {
 		$gitversion = Config::get('system', 'git_friendica_version');
 		if (version_compare(FRIENDICA_VERSION, $gitversion) < 0) {
-			$warningtext[] = sprintf(t('There is a new version of Friendica available for download. Your current version is %1$s, upstream version is %2$s'), $FRIENDICA_VERSION, $gitversion);
+			$warningtext[] = sprintf(t('There is a new version of Friendica available for download. Your current version is %1$s, upstream version is %2$s'), FRIENDICA_VERSION, $gitversion);
```

Comment:
```t()``` includes ```sprintf()``` doesn't it?

Reply:
Yep, I'll simplify the call. I'm not concentrating on it because it isn't a code error per se.